### PR TITLE
Added Redbee Media to Publisher class

### PR DESCRIPTION
**Pull request by timothy22000**

### CODE REVIEW

**Comment by JamieAP** [comment]
tim_wip tells me nothing about what Im reviewing ;)

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
```
Comment (Dalamar42) [inline .gitignore]: This should be in your global .gitignore, not here

File: src/main/java/org/atlasapi/media/entity/Publisher.java [hunk 2]
```diff
@@ -198,7 +198,8 @@
     BT_SPORT_ZEUS("BT Sport Zeus", "zeus.sport.bt.com", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false),
     BT_SPORT_DANTE("BT Sport Dante", "dante.sport.bt.com", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false),
     UKTV("UKTV", "uktv.co.uk", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false),
-    BT_SPORT_EBS("BT Sport EBS", "ebs.sport.bt.com", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false);
+    BT_SPORT_EBS("BT Sport EBS", "ebs.sport.bt.com", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false),
+    REDBEE_MEDIA("Redbee Media", "redbeemedia.com", Countries.GB, SourceStatus.ENABLEABLE, SourceRestriction.ADMIN_ONLY, false);
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/media/entity/Publisher.java]: Looks good

**Comment by Dalamar42** [comment]
It's generally useful to name branches after what they do :)

**Comment by timothy22000** [comment]
Thanks Jamie and Thomas. :D

**Pull request closed without merging**